Automattic/wp-calypso
# Remove the 1y/2y term experiment code

Fix Automattic/martech#1709

## Proposed Changes

As the 1y/2y term experiment has been concluded(see p2-pbxNRc-2qr#comment-4418), this PR cleans up the experiment code introduced by #75063.

Note that `calypso_plans_plan_type_selector_view` event is intentionally kept. We've registered this event and it doesn't seem to hurt.

## Testing Instructions

* The term toggle in the sign-up flow and the plans administration page should work as expected.
* Previously there is a loading placeholder for the term toggle. Now that the experiment is removed, it should just load.
* The checkout page should work as expected. Notebly, you should only be able to choose higher term in the checkout page.

## Pre-merge Checklist
